public/app/features/dashboard/state/PanelQueryRunner.ts

```diff
@@ -66,37 +75,44 @@ export class PanelQueryRunner {
   /**
    * Returns an observable that subscribes to the shared multi-cast subject (that reply last result).
    */
-  getData(transform = true): Observable<PanelData> {
+  getData(options?: GetDataOptions): Observable<PanelData> {
+    const { transform, applyFieldConfig } = options
+      ? {
+          ...DEFAULT_GET_DATA_OPTIONS,
+          ...options,
+        }
+      : DEFAULT_GET_DATA_OPTIONS;
```

Comment:
can't we do getData(options? = DEFAULT_GET_DATA_OPTIONS), might be good require options & the bool flags, to make this more explicit so we do not have more bugs with wrong data.